spec/fixtures/_nodes/7x.json

```diff
@@ -20,7 +20,6 @@
       ],
       "attributes" : {
         "ml.machine_memory" : "17179869184",
```

Comment:
Cant we remove all ml related code as its xpack?

Reply:
Ack